vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -14,196 +14,212 @@ spec:
     listKind: VerticalPodAutoscalerCheckpointList
     plural: verticalpodautoscalercheckpoints
     shortNames:
-    - vpacheckpoint
+      - vpacheckpoint
```

Comment:
Please don't change formatting in this file.

Reply:
Thanks, the formatting issue has been fixed.

Reply:
> Please don't change formatting in this file.

Thanks, the formatting issue has been fixed.